backend/src/main/java/com/senior/project/backend/Activity/MilestoneRepository.java

```diff
@@ -16,36 +18,79 @@ public class MilestoneRepository {
     static {
         Task task1 = Task.builder()
                 .id("Task 1 id")
```

Comment:
any reason why the ids are strings and not ints? wouldn't it be easier to request a task by an int id?

Reply:
this also goes for event ids and milestone ids. both of those have string name fields as well as string ids. why list "event 1 id" and not just 1?

Reply:
we don't have a database yet, so we don't have a defined way of storing IDs, but usually storing UUIDs as strings is better

Reply:
i had most of the api calls using ids as ints. should i change that or could we use the uuids as ints even if we store them as strings or smth?

Reply:
I would say let's figure out what exactly Spring supports for MySQL because it likely has a built-in way to generate the IDs we can just use. I know you can Create ObjectIds which are like UUIDS and just long random strings, but that was for MongoDB. I'm not sure what it would look like for MySQL

Reply:
okay good to know